packages/dds/task-manager/src/taskManager.ts

```diff
@@ -312,6 +312,34 @@ export class TaskManager extends SharedObject<ITaskManagerEvents> implements ITa
         return lockAcquireP;
     }
 
+    public subscribeToTask(taskId: string) {
+        const disconnectHandler = () => {
+            // Wait to be connected again and then re-submit volunteer op
+            this.connectionWatcher.once("connect", () => {
```

Comment:
Currently there's no way to abandon in disconnected state (it throws), but this was partly because it didn't make sense in a pre-subscription world.  With subscription, maybe I want to be able to abandon while disconnected, to prevent volunteering upon reconnect?  If so, this handler would need to also be removed in response to abandonWatcher.